core/libp2p/connection/capable_connection.hpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CAPABLE_CONNECTION_HPP
+#define KAGOME_CAPABLE_CONNECTION_HPP
+
+#include "libp2p/connection/multiaddr_connection.hpp"
+#include "libp2p/connection/raw_connection.hpp"
+#include "libp2p/connection/secure_connection.hpp"
+#include "libp2p/connection/stream.hpp"
+
+namespace libp2p::connection {
+
+  // connection that provides basic libp2p requirements to the connection
```

Comment:
I believe it would be more consistent to comment it just like in other places:
```suggestion
/**
   * @brief Connection that provides basic libp2p requirements to the connection.
   */
```

Reply:
I heard that we can use /// for single line comments

Reply:
in tasks related to implementation of those interfaces implementers should change // comments to /**/ comments themselves. here it is just a short hint, what actually this class does